.buffy/templates/reviewer_checklist.md

```diff
@@ -14,6 +14,10 @@
 - [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
 - [ ] **Contribution and authorship:** Has the submitting author ({{author-handle}}) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
 - [ ] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)
+- [ ] **Data sharing:** If the paper contains original data, data are accessible to the reviewers.
+- [ ] **Data sharing:** If the paper contains original results, results are entirely reproducible by reviewers
```

Comment:
The heading here is wrong - maybe this should be reproducibility?  Also, I don't think "entirely" is always possible.  Finally, perhaps add "If the paper contains no original results, please check this item"

Reply:
Thanks @danielskatz! Fixing now.